pkg/config/linters.go

```diff
@@ -31,3 +38,27 @@ func (l *Linters) Validate() error {
 
 	return nil
 }
+
+func (l *Linters) validateNoFormattersEnabled() error {
+	for _, n := range l.Enable {
+		if slices.Contains(getAllFormatterNames(), n) {
+			return fmt.Errorf("%s is a formatter", n)
+		}
+	}
+
+	return nil
+}
+
+func (l *Linters) validateNoFormattersDisabled() error {
+	for _, n := range l.Enable {
```

Comment:
Perhaps this should be `Disable` slice:
```suggestion
	for _, n := range l.Disable {
```